libs/core/input-group/input-group.component.html

```diff
@@ -81,5 +81,6 @@
         [attr.placeholder]="_cvaControl.cvaDirective?.placeholder || null"
         [attr.aria-required]="required"
         (search)="_onSearchEvent($event)"
+        autocomplete="off"
```

Comment:
Is it necessary for the input group component to always have autocomplete set to "off"? @mikerodonnell89

Reply:
You're right, probably best to just do this specifically for combobox/multi-input etc